backend/src/org/commcare/session/CommCareSession.java

```diff
@@ -52,7 +54,7 @@
     /**
      * A table of all datums (id --> value) that are currently on the session stack
      */
-    private final OrderedHashtable collectedDatums;
+    private final OrderedHashtable<String, String> collectedDatums;
```

Comment:
Adding type annotations like this help prevent runtime errors by doing type checking during compile time.
And they prevent annoying lint warnings :)